src/components/tagline.jsx

```diff
@@ -5,20 +5,24 @@ import PropTypes from 'prop-types';
 
 function Tagline({ tagline }) {
   return (
-    <GridContainer className="usa-section">
-      <Grid row gap>
-        {tagline.heading && (
-          <Grid tablet={{ col: 4 }}>
-            <h2 className="font-heading-xl">{tagline.heading}</h2>
-          </Grid>
-        )}
-        {tagline.text && (
-          <Grid className="usa-prose" tablet={{ col: 8 }}>
-            <p>{tagline.text}</p>
-          </Grid>
-        )}
-      </Grid>
-    </GridContainer>
+    <section className="usa-section">
+      <GridContainer className="usa-section">
```

Comment:
Lines 8 and 9 utilize the same class.

Reply:
Oh thanks - forgot to remove from the grid container when I added sections!